elastic-agent-client.proto

```diff
@@ -107,6 +107,12 @@ enum UnitLogLevel {
     TRACE = 4;
 }
 
+// Reports the mode agent is running in
+enum AgentManagedMode {
+    FLEET = 0;
+    SELFMANAGED = 1;
```

Comment:
I don't know if these values are being used somewhere already — hopefully not — but it would be good to name them consistently with existing verbiage for these concepts.
```suggestion
    MANAGED = 0;
    STANDALONE = 1;
```